tests/scripts/test_psa_constant_names.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, filename, line_number):
         self.line_number = line_number
 
 class read_file_lines:
+    # Dear Pylint, conventionally, a context manager class name is lowercase.
```

Comment:
Did you consider using https://docs.python.org/3/library/contextlib.html#contextlib.contextmanager here?

Reply:
Would it help? This context manager doesn't the acquire/release pattern that is the typical use case of `contextlib.contextmanager`.

Reply:
I presumed that could have silence the code style error, but I have just check, and it doesn't, so no, unfortunately it would not help. Please disregard my above comment.